### DISCUSSION TITLE
Frigate integration

### DISCUSSION THREAD

**Discussion by jberg145** [blakeblackshear/frigate#380] [General]
First i want to start by saying awesome job on Frigate.

Is there a specific version of home assistant required for the frigate integration?  I am running HA version .115.4 and loaded the custom component and restarted HA but do not see it in the integrations section.  

In the logs the following entry is the only thing i see so it seems like it is loading it. 
WARNING (MainThread) [homeassistant.loader] You are using a custom integration for frigate which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.

**Comment by blakeblackshear** [comment]
You have to go to integrations click add and search for frigate.

**Reply by jberg145** [reply]
Thanks, thought i checked there...guess not.  Thanks again for the help.
